### ISSUE TITLE
Syntax Warnings for Invalid Escape Sequence

### ISSUE THREAD

**Issue by danyeaw** [harfbuzz/harfbuzz#4682]
I think fix_get_types.py should be using a raw string so that \( isn't read as an escaped character. I can submit a PR.

```
C:\gtk-build\build\x64\release\harfbuzz\src\fix_get_types.py:14: SyntaxWarning: invalid escape sequence '\('
  l = re.sub ('_T \(', ' (', l)
[59/251] Generating src/hb-gobject-enums.cc with a custom command
C:\gtk-build\build\x64\release\harfbuzz\src\fix_get_types.py:14: SyntaxWarning: invalid escape sequence '\('
  l = re.sub ('_T \(', ' (', l)
```

**Comment by behdad** [comment]
Please do. Though how has this been working? Are you using Python 2 by any chance?

**Comment by danyeaw** [comment]
I am using Python 3.12.2. This is a change in [Python 3.12](https://docs.python.org/3/whatsnew/3.12.html):

A backslash-character pair that is not a valid escape sequence now generates a [SyntaxWarning](https://docs.python.org/3/library/exceptions.html#SyntaxWarning), instead of [DeprecationWarning](https://docs.python.org/3/library/exceptions.html#DeprecationWarning). For example, re.compile("\d+\.\d+") now emits a [SyntaxWarning](https://docs.python.org/3/library/exceptions.html#SyntaxWarning) ("\d" is an invalid escape sequence, use raw strings for regular expression: re.compile(r"\d+\.\d+")). In a future Python version, [SyntaxError](https://docs.python.org/3/library/exceptions.html#SyntaxError) will eventually be raised, instead of [SyntaxWarning](https://docs.python.org/3/library/exceptions.html#SyntaxWarning). (Contributed by Victor Stinner in [gh-98401](https://github.com/python/cpython/issues/98401).)

**Comment by danyeaw** [comment]
This was already fixed in 2bd6124737c71df248c6b3af1d201aab93acc428. Sorry for the noise.

**Comment by behdad** [comment]
No worries. Thanks.
